.circleci/config.yml

```diff
@@ -254,6 +278,40 @@ workflows:
             branches: { ignore: /docs\-.*/ }
       - react_test: {requires: [react_build_storybook]}
 
+  # Release branch automation every Tuesday at 1 p.m. ET
+  release:
+    jobs:
```

Comment:
do these workflows start asynchronously? I feel like this task should only run after `build_branch` on develop not just `patternlab_build` right?